config/src/config/storage_config.rs

```diff
@@ -14,6 +14,8 @@ pub struct StorageConfig {
     pub simple_address: SocketAddr,
     pub dir: PathBuf,
     pub grpc_max_receive_len: Option<i32>,
+    // None means no pruning.
```

Comment:
use triple /// None disables pruning.

Could you share a formula here for computing windows?